pkg/endpoint/endpoint.go

```diff
@@ -815,6 +815,16 @@ func (e *Endpoint) GetLabels() []string {
 	return e.SecurityIdentity.Labels.GetModel()
 }
 
+// GetSecurityIdentity returns the security identity of the endpoint.
+func (e *Endpoint) GetSecurityIdentity() *identityPkg.Identity {
```

Comment:
Does this assume the endpoint is locked?

Reply:
Yes, I will add a comment indicating as such.